libbpf-tools/memleak.c

```diff
@@ -88,8 +90,13 @@ struct allocation {
 
 #define __ATTACH_UPROBE(skel, sym_name, prog_name, is_retprobe) \
 	do { \
+		char sym[32]; \
+		if (env.sym_pre) \
```

Comment:
This is always `true`, right ? I think we don't need the `else` brance below.

Reply:
done